harpy/bin/inline_to_haplotag.py

```
@@ -52,27 +53,41 @@
+        _new_fw += fw_entry[3][bc_len:] + "\n"
+        if rv_entry:
+            _new_rv  = rv_entry[0].split()[0] + f"\tOX:Z:{bc_inline}\tBX:Z:{bc_hap}\n"
+            _new_rv += rv_entry[1] + "\n"
+            _new_rv += rv_entry[2] + "\n"
+            _new_rv += rv_entry[3] + "\n"
+        else:
+            _new_rv = None
+        return _new_fw, _new_rv
+    else:
+        # no forward read, therefor no barcode to search for
+        _new_rv  = rv_entry[0].split()[0] + f"\tBX:Z:A00C00B00D00\n"
```

Ruff

f-string without any placeholders

Remove extraneous `f` prefix

(F541)